<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCustomer</class>
 <widget class="QMainWindow" name="NewCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>AJ Bank | New Customer Registration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/background/NewCustomer.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CreateUsernameB">
    <property name="geometry">
     <rect>
      <x>966</x>
      <y>408</y>
      <width>328</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/User.png);
background-position:center;border-radius:100%;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="NewNameT">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/White.png);font: 63 18pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NewEmailIDT">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/White.png);font: 63 18pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NewAddressT">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>550</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/White.png);font: 63 18pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NewPhone1T">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/White.png);font: 63 18pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
   </widget>
   <widget class="QDateEdit" name="NewDOB">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="NewGender">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/White.png);</string>
    </property>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Other</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>93</y>
      <width>150</width>
      <height>581</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Back.png);
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SuggestButton">
    <property name="geometry">
     <rect>
      <x>949</x>
      <y>0</y>
      <width>421</width>
      <height>384</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Suggest.png);
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorMessage">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>630</y>
      <width>841</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Blue.png);
color: rgb(255, 255, 255);
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>CreateUsernameB</zorder>
   <zorder>BackButton</zorder>
   <zorder>SuggestButton</zorder>
   <zorder>NewAddressT</zorder>
   <zorder>NewPhone1T</zorder>
   <zorder>NewDOB</zorder>
   <zorder>NewEmailIDT</zorder>
   <zorder>NewGender</zorder>
   <zorder>NewNameT</zorder>
   <zorder>ErrorMessage</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
